spark-connect/client/src/test/scala-spark-master/io/delta/connect/tables/RemoteSparkSession.scala

```diff
@@ -97,10 +98,14 @@ trait RemoteSparkSession extends BeforeAndAfterAll { self: Suite =>
   override def beforeAll(): Unit = {
     super.beforeAll()
     server
-    // TODO: Instead of sleeping for a fixed time, which is a bit brittle,
-    // we should repeatedly check when the server is ready.
-    Thread.sleep(10000)
-    spark = SparkSession.builder().remote(s"sc://localhost:$serverPort").build()
+
+    val sparkConnectClient = SparkConnectClient
+      .builder()
+      .userId("test")
+      .port(serverPort)
+      .retryPolicy(RetryPolicy.defaultPolicy())
```

Comment:
I don't think this is necessary. This is already the default.

Reply:
@tomvanbussel I put it here because I feel like the fact that is we are using the `defaultPolicy` might not be obvious and the viewers might not know, should I put a comment instead?

Reply:
Unfortunately it requires accessing a package private interface. I would rather use the public interfaces, even if that means not being explicit about using the default retry policy.